react_app/package.json

```diff
@@ -16,17 +16,22 @@
     "react-portal": "^4.2.2",
     "react-router-dom": "^6.10.0",
     "react-scripts": "5.0.1",
+    "ts-loader": "^9.4.2",
     "uuidv4": "^6.2.13",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "webpack --mode production",
+    "start": "webpack serve --open --mode development",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint ./src --fix",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "cypress:open": "cypress open",
     "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "prepare": "cd .. && husky install react_app/.husky",
+    "pre-commit": "lint-staged"
```

Comment:
When someone else tries to contribute to your repository, they will get the following error:
```cmd
hint: The 'react_app/.husky/pre-commit' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
```
To make hook executable, run these commands:
```cmd
> chmod ug+x .husky/*
> chmod ug+x ../.git/hooks/*
```
It's better to leave this info into `readmy.md`